samples/model-builder/create_training_pipeline_image_classification_sample_test.py

```diff
@@ -44,7 +44,9 @@ def test_create_training_pipeline_image_classification_sample(
         project=constants.PROJECT, location=constants.LOCATION
     )
     mock_get_automl_image_training_job.assert_called_once_with(
-        display_name=constants.DISPLAY_NAME
+        display_name=constants.DISPLAY_NAME,
+        multi_label=False, 
```

Comment:
lint: complaining about trailing whitespace
```suggestion
        multi_label=False,
```

Reply:
fixed